src/schema/rules/sidecars/mri.yaml

```diff
@@ -67,6 +69,10 @@ MRISequenceSpecifics:
     SpoilingRFPhaseIncrement: optional
     SpoilingGradientMoment: optional
     SpoilingGradientDuration: optional
+    WaterSuppression: optional
+    WaterSuppressionTechnique: optional
```

Comment:
So do we want to add something like the following to `WaterSuppression`:

> Corresponds to DICOM Tag 0018, 9025 `SpectrallySelectedSuppression`.
> If present and not `NONE`, then this value SHOULD be True, otherwise False.

And the following to `WaterSuppressionTechnique`?

> Corresponds to DICOM Tag 0018, 9025 `SpectrallySelectedSuppression`.
> May be omitted if the value is `NONE`.

Reply:
Makes sense to me! And the same would be the case for `OuterVolumeSuppression`.